messaging-experiments.yaml

```diff
@@ -200,3 +200,118 @@
+          value: null
+      - slug: msw-default
+        ratio: 1
+        feature:
+          featureId: "aboutwelcome"
+          enabled: true
+          value: {}
+      - slug: msw-no-titles
+        ratio: 1
+        feature:
+          featureId: "aboutwelcome"
+          enabled: true
+          value:
+            id: msw-no-titles
+            template: multistage
+            screens:
+              - id: "AW_GET_STARTED"
+                order: 0
+                content:
+                  zap: true
+                  title: Welcome to Firefox
+                  subtitle: The fast, safe, and private browser that's backed by a non-profit.
+                  primary_button:
+                    label: Start Setup
+                    action:
+                      navigate: true
+                  secondary_button:
+                    text: Have an account?
+                    label: Sign in
+                    position: top
+                    action:
+                      type: OPEN_URL
+                      addFlowParams: true
+                      data:
+                        args: https://accounts.firefox.com/?service=sync&action=email&context=fx_desktop_v3&entrypoint=activity-stream-firstrun&style=trailhead
+                        where: current
+              - id: "AW_IMPORT_SETTINGS"
+                order: 1
+                content:
+                  zap: true
+                  title: Import your passwords, bookmarks, and more
+                  subtitle: Coming from another browser? It's easy to bring everything to Firefox.
+                  tiles:
+                    type: topsites
+                    showTitles: false
+                  primary_button:
+                    label: Start Import
+                    action:
+                      navigate: true
+                      type: SHOW_MIGRATION_WIZARD
+                  secondary_button:
+                    label: Not now
+                    action:
+                      navigate: true
+              - id: "AW_CHOOSE_THEME"
+                order: 2
+                content:
+                  zap: true
+                  title: Choose a look
+                  subtitle: Personalize Firefox with a theme.
+                  tiles:
+                    type: theme
+                    action:
+                      theme: <event>
+                    data:
+                      - theme: automatic
+                        label: Automatic
+                        tooltip:
+                          string_id: onboarding-multistage-theme-tooltip-automatic
+                      - theme: light
+                        label: Light
+                        tooltip:
+                          string_id: onboarding-multistage-theme-tooltip-light
+                      - theme: dark
+                        label: Dark
+                        tooltip:
+                          string_id: onboarding-multistage-theme-tooltip-dark
```

Comment:
Its missing Alpenglow, will be good to compare JSON with https://searchfox.org/mozilla-central/source/browser/components/newtab/content-src/lib/aboutwelcome-utils.js